exercise/admin.py

```diff
@@ -94,22 +94,50 @@ def has_change_permission(self, request, obj=None):
         return super().has_change_permission(request, obj)
 
 
+class ExerciseEmailPropertiesListFilter(admin.SimpleListFilter):
+    title = "Exercise"
+    parameter_name = "exercise"
+
+    def lookups(self, request, model_admin):
+        exercises = Exercise.user_objects.filter_by_user(request.user)
```

Comment:
There's a request in #328, so once that new method is available in the manager let's switch to that.
Because with just `.exclude(organisation=None)` superadmins won't be able to change email properties.

Reply:
Nice. But are we addressing `ExerciseQuerySet` itself in that request? If not, then I think we have to, as that is main queryset for this.

Reply:
yes that should be added to `ExerciseQuerySet`